calico/network-policy/encrypt-cluster-pod-traffic.mdx

```diff
@@ -255,7 +255,7 @@ calicoctl patch felixconfiguration node.my-node --type='merge' -p '{"spec":{"wir
 To verify that the nodes are configured for WireGuard encryption, check the node status set by Felix using `calicoctl`. For example:
 
 ```bash
```

Comment:
Is the indentation here intentional?

Reply:
It was intentional. Either ways, it's fine. Reverted.